packages/administration/src/Component/Action/RowAction.php

```diff
@@ -89,6 +114,18 @@ public function validate(mixed $data): bool
             throw new InvalidArgumentException('Route must be set for row action. Use one of the "linkTo*" methods.');
         }
 
+        // Evaluate disabled message callback and modify action state if disabled
+        if ($this->disabledMessageCallback !== null) {
```

Comment:
IMHO the logic does not fit into the `validate()` method :thinking: I am wondering, where to put it. I thought about `create` but that would not work because it is static, right? Let's talk about it, please

Reply:
I refactored it so now its more encapsulated. What do you think?